<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Salvar para concluir mais tarde</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Concluir a verificação</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Dinheiro no caixa</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelNotas">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Notas</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelR2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelR5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 5</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="labelR10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 10</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="labelR20">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 20</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="labelR50">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 50</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="labelR100">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 100</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLabel" name="labelTotalBills__">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Total</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinR2"/>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="spinR5"/>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="spinR10"/>
          </item>
          <item row="1" column="4">
           <widget class="QSpinBox" name="spinR20"/>
          </item>
          <item row="1" column="5">
           <widget class="QSpinBox" name="spinR50"/>
          </item>
          <item row="1" column="6">
           <widget class="QSpinBox" name="spinR100"/>
          </item>
          <item row="1" column="7">
           <widget class="QLabel" name="labelTotalBills">
            <property name="text">
             <string>R$ 0,00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelMoedas">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Moedas</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="labelR001">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 0,01</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="labelR005">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 0,05</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="labelR010">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 0,10</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="labelR025">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 0,25</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="labelR050">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 0,50</string>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <widget class="QLabel" name="labelR1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>R$ 1,00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="7">
           <widget class="QLabel" name="labelTotalCoins__">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Total</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinR001"/>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="spinR005"/>
          </item>
          <item row="3" column="3">
           <widget class="QSpinBox" name="spinR010"/>
          </item>
          <item row="3" column="4">
           <widget class="QSpinBox" name="spinR025"/>
          </item>
          <item row="3" column="5">
           <widget class="QSpinBox" name="spinR050"/>
          </item>
          <item row="3" column="6">
           <widget class="QSpinBox" name="spinR1"/>
          </item>
          <item row="3" column="7">
           <widget class="QLabel" name="labelTotalCoins">
            <property name="text">
             <string>R$ 0,00</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Retiradas e depósitos no perído</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <property name="cornerButtonEnabled">
             <bool>true</bool>
            </property>
            <property name="rowCount">
             <number>0</number>
            </property>
            <property name="columnCount">
             <number>4</number>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Horário</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Responsável</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Valor</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Descrição</string>
             </property>
            </column>
           </widget>
          </item>
          <item alignment="Qt::AlignTop">
           <widget class="QToolButton" name="addDepositOrWithdraw">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="7" column="1">
           <widget class="QLabel" name="caixaEsperado">
            <property name="text">
             <string>R$ 0,00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="retiradasPeriodo">
            <property name="text">
             <string>R$ 0,00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Saldo de retiradas e depósitos</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Total esperado no caixa</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="vendasDinheiroPeriodoAnterior">
            <property name="text">
             <string>R$ 0,00</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Contagem de cédulas e moedas no caixa</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Dinheiro no caixa no início do período anterior</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Vendas em dinheiro deste período</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="totalCaixaPeriodo">
            <property name="text">
             <string>R$ 0,00</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="subtotalCaixa">
            <property name="text">
             <string>R$ 0,00</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="vendasDinheiroPeriodo">
            <property name="text">
             <string>R$ 0,00</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_31">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Contagem do caixa</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Apuração do caixa</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Total no caixa no periodo</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Diferença no caixa</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="diferencaCaixa">
            <property name="text">
             <string>R$ 0,00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Cartões</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Totais do período</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <row>
           <property name="text">
            <string>POS</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>TEF</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Totais</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Mastercard</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Maestro</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Visa</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Visa Electron</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>AMEX</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ELO</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Diners/Hipercard</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Visa 2x</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Visa 3x</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Mastercard 2x</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Mastercard 3x</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>AMEX 2x</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>AMEX3x</string>
           </property>
          </column>
          <item row="0" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>Conferência do POS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string>Bandeira</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Modalidade</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Valor</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>AUTO</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>AUTE</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Conferência</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Observações</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinR2</tabstop>
  <tabstop>spinR5</tabstop>
  <tabstop>spinR10</tabstop>
  <tabstop>spinR20</tabstop>
  <tabstop>spinR50</tabstop>
  <tabstop>spinR100</tabstop>
  <tabstop>spinR001</tabstop>
  <tabstop>spinR005</tabstop>
  <tabstop>spinR010</tabstop>
  <tabstop>spinR025</tabstop>
  <tabstop>spinR050</tabstop>
  <tabstop>spinR1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>spinR2Changed(int)</slot>
 </slots>
</ui>
